js/Track/Model/File/WIG.js

```diff
@@ -0,0 +1,40 @@
+Genoverse.Track.Model.File.WIG = Genoverse.Track.Model.Graph.Bar.extend({
+  dataType : 'text',
+  getData : function(chr, start, end){
+    if(!this.url){
+      this.isLocal = true;
+      this.dataFile = this.track.dataFile;
+      Genoverse.Track.Model.File.prototype.getData.apply(this, arguments);
+    }
+    return this.base.apply(this, arguments);
+  },
+  parseData : function(text, chr, s, e){
+    var lines = text.split('\n');
+    var chrom, start, step, span;
+    var features = [];
+
+    for(var i = 0; i < lines.length; i++){
+      var fields = lines[i].split(' ');
+
+      if(fields[0].indexOf('#') != -1 || fields[0] == 'track') continue;
+
+      if(fields[0] == 'fixedStep'){
```

Comment:
It would be more efficient to put this check for fixedStep outside of the for loop:

```
var line;

while (lines.length && !/chrom/.test(line)) {
  line = lines.shift();
}

if (line) {
  // assign chrom, start, etc. based on line
}

for(var i = 0; i < lines.length; i++) {
  ...
}
```

Also, we need to be able to deal with variableStep files.